rero_ils/modules/patrons/api.py

```diff
@@ -448,24 +447,13 @@ def can_request(cls, item, **kwargs):
 
     @classmethod
     def can_checkout(cls, item, **kwargs):
-        """Check if a patron can checkout an item.
+        """Check if a patron can checkout an item."""
+        return cls.can_request(item, **kwargs)
```

Comment:
maybe you can explain in comment why `can_checkout` returns result of `can_request` or create a new function based on `can_request` logic